Lib/uuid.py

```diff
@@ -322,8 +335,9 @@ def clock_seq_low(self):
 
     @property
     def time(self):
-        return (((self.time_hi_version & 0x0fff) << 48) |
-                (self.time_mid << 32) | self.time_low)
+        if self.version == 6:
```

Comment:
Shouldn't there be if clauses for v7 and v8 as well?
For example, this is how third party [uuid6](https://github.com/oittaa/uuid6-python/tree/main) package is implemented:
https://github.com/oittaa/uuid6-python/blob/4f879849178b8a7a564f7cb76c3f7a6e5228d9ed/src/uuid6/__init__.py#L59-L70

Reply:
Yes, that's what I'm wondering about. I think in the end I'll remove that altogether, but I was too lazy to remove it for now.

Reply:
Note that most implementations might also be based on old RFC editions for which the specifications were not entirely clear.